### PR TITLE
Overwatch Skill + All USCM Officers Can Now Overwatch

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

Separates overwatch console usage out of the leadership skill and into its own skill, the Overwatch skill. 

Grants all USCM officers the ability to use overwatch consoles, for low-pop hours. 

# Explain why it's good for the game

Giving people access to overwatch was constrained by the fact that the minimum leadership skill to use OW gave you a high rank order, this change now allows you to give people OW access without giving them access to strong orders. 

Furthermore, this now allows all officers the ability to use the OW console. This should help for low-pop hours or for when a officer wants to check the OW for information. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: A new skill has been added, the 'Overwatch' skill, allows the user to use the overwatch console.
add: All USCM Officers can now use overwatch consoles. 
/:cl:

### CODE REVIEW

File: code/datums/skills.dm [hunk 1]
```diff
@@ -795,6 +816,7 @@ MILITARY NONCOMBATANT
 		SKILL_PILOT = SKILL_PILOT_TRAINED,
 		SKILL_POWERLOADER = SKILL_POWERLOADER_MASTER,
 		SKILL_LEADERSHIP = SKILL_LEAD_TRAINED,
+		SKILL_OVERWATCH = SKILL_OVERWATCH_TRAINED,
```
Comment (morrowwolf) [inline code/datums/skills.dm:819]: Not DCC

File: code/datums/skills.dm [hunk 2]
```diff
@@ -775,6 +794,7 @@ MILITARY NONCOMBATANT
 		SKILL_SURGERY = SKILL_SURGERY_TRAINED,
 		SKILL_RESEARCH = SKILL_RESEARCH_TRAINED,
 		SKILL_INTEL = SKILL_INTEL_TRAINED,
+		SKILL_OVERWATCH = SKILL_OVERWATCH_TRAINED,
```
Comment (morrowwolf) [inline code/datums/skills.dm:797]: Not researcher

File: code/datums/skills.dm [hunk 3]
```diff
@@ -757,6 +775,7 @@ MILITARY NONCOMBATANT
 		SKILL_FIREARMS = SKILL_FIREARMS_CIVILIAN,
 		SKILL_MEDICAL = SKILL_MEDICAL_DOCTOR,
 		SKILL_SURGERY = SKILL_SURGERY_TRAINED,
+		SKILL_OVERWATCH = SKILL_OVERWATCH_TRAINED,
```
Comment (morrowwolf) [inline code/datums/skills.dm:778]: Not doctor

**Review by morrowwolf — CHANGES_REQUESTED** [review]

**Comment by Steelpoint** [comment]
Requested changes made

**Review by morrowwolf — APPROVED** [review]

**Pull request merged**